requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
--r requirements.txt
 coverage==4.2
 django_coverage_plugin==1.3.1
 mock==1.0.1
+ipdb
```

Comment:
I don't see a big need to not have all requirements installed. In the end it's a few megabytes that won't be imported anyway